packages/types/src/interfaces/tracer.ts

```diff
@@ -32,11 +32,11 @@ export interface Tracer {
   currentSpan(): NodeSpan
 
   /**
+   *
    * Returns the root Span.
    *
-   * If there is no root Span available, `undefined` is returned.
    */
-  rootSpan(): NodeSpan | undefined
+  rootSpan(): NodeSpan
```

Comment:
Could it still return undefined if no span is started? Or would it then return a `NoopSpan`?

If it returns a `NoopSpan` I would change the changeset to something like the message below to explain the impact of the change.

> Tracer `rootSpan` does not return undefined anymore, but returns a `NoopSpan` instead.

What is the accompanying PR for Node.js that implements this change?

Reply:
It'll return a `NoopSpan`, yes. 

I'm not sure if we should note that the `NoopSpan` is going to be returned here as it is an implementation detail of the NodeJS library and it stills comply with the interface. In NodeJS library, `NoopSpan` is an implementation of `NodeSpan`.

I haven't opened the PR using these changes yet as I'm still testing the integrations and I'd also like to update all the dependencies on the NodeJS library with this new version to avoid build problems. Because new NodeJS changes won't work if the types package is below this new version.

Reply:
Changesets are not only a list of technical changes. The updated message would help to explain the change to users of the library. Now it doesn't tell the user how it would impact them. Like how we mentioned in the removed comment when it would return `undefined` in some scenarios.

For local development you don't have to wait until this is published, you can link the package locally so it uses your local package.

Reply:
I understand, but users of this library might not be users of the NodeJS library, so talking about implementation details or usages on its dependent libraries seems misleading to me. Anyway, I updated it!

> For local development, you don't have to wait until this is published. You can link the package locally, so it uses your local package.

I know, I did the link for my tests on the integrations, but I'd like to consider it finished and ready to review once these changes are released.